### PR TITLE
feat(video): add autoPlay and loop options

### PR DESCRIPTION
**Pull request by ana-oprea**
…kground

### CODE REVIEW

File: src/NextCloud/players/NextCloudEmbed.jsx [hunk 1]
```diff
@@ -22,7 +23,9 @@ const NextCloudEmbed = ({ data, embedSettings }) => {
             : `${flattenToAppURL(data.url)}/@@download/file`
           : `${data.url}/download`
       }
-      controls
+      controls={!data.use_video_as_background}
```
Comment (avoinea) [inline src/NextCloud/players/NextCloudEmbed.jsx]: Please use explicit options for `autoPlay` and `loop`
```
controls={!data.autoPlay}
autoPlay={data.autoPlay}
loop={data.loop}
```

File: src/NextCloud/schema.js [hunk 2]
```diff
@@ -17,6 +17,10 @@ const messages = defineMessages({
     id: 'Alignment',
     defaultMessage: 'Alignment',
   },
+  use_video_as_background: {
```
Comment (avoinea) [inline src/NextCloud/schema.js]: Please replace with:
- autoPlay
- loop

File: src/NextCloud/schema.js [hunk 3]
```diff
@@ -25,7 +29,7 @@ export const VideoBlockSchema = (props) => ({
     {
       id: 'default',
       title: 'Default',
-      fields: ['url', 'preview_image', 'align'],
+      fields: ['url', 'preview_image', 'align', 'use_video_as_background'],
```
Comment (avoinea) [inline src/NextCloud/schema.js]: `use_video_as_background` to be replaced with `autoPlay` and `loop`

File: src/NextCloud/players/NextCloudEmbed.jsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/media-has-caption */
```
Comment (avoinea) [inline src/NextCloud/players/NextCloudEmbed.jsx:1]: See https://taskman.eionet.europa.eu/issues/152940#note-29

**Review by avoinea — CHANGES_REQUESTED** [review]

File: src/NextCloud/players/NextCloudEmbed.jsx (hunk 1 above)
Comment (ana-oprea) [reply src/NextCloud/players/NextCloudEmbed.jsx]: resolved

File: src/NextCloud/players/NextCloudEmbed.jsx (hunk 4 above)
Comment (ana-oprea) [reply src/NextCloud/players/NextCloudEmbed.jsx:1]: to be added later

**Review by avoinea — APPROVED** [review]

**Pull request merged**